packages/helm-charts/celo-fullnode/templates/service.yaml

```diff
@@ -0,0 +1,22 @@
+{{ range $index, $e := until (int .Values.replicaCount ) }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "celo-fullnode.fullname" $ }}-{{ $index }}
+  labels:
+{{ include "standard.labels" $  | indent 4 }}
+spec:
+  type: {{ $.Values.service.type }}
+{{- if $.Values.geth.use_static_ips }}
+  loadBalancerIP: {{ index $.Values.geth.public_ips $index -}}
+{{- end }}
+  ports:
+    - port: 30303
```

Comment:
I know we talked about this directly, but we should add UDP support as well to accommodate discovery